### ISSUE TITLE
new-line-before-expect improvement

### ISSUE THREAD

**Issue by surdu** [tlvince/eslint-plugin-jasmine#141]
The following example I feel it should not trigger a `new-line-before-expect` warning:

```
it("should do something", function () {
  var a = 3;

  expect(a).toBeDefined();
  expect(a).toBe(3);
});
```

**Comment by burabure** [comment]
@DianaSuvorova I think this is valid improvement. what do you think?

**Comment by DianaSuvorova** [comment]
Yes, good suggestion! Will do.

**Comment by surdu** [comment]
@DianaSuvorova May I have a try at fixing this?

**Comment by DianaSuvorova** [comment]
@surdu , Yes, of course! You are now officially assigned to the ticket 🙂
